lib/pbench/server/database/models/users.py

```diff
@@ -112,14 +124,12 @@ def delete(username):
             raise
 
     def is_admin(self):
-        # TODO: Add notion of Admin user
-        """this method would always return false for now until we add a notion of Admin user/group.
-        Once we know the admin credentials this method can check against those credentials to determine
-        whether the user is privileged to do more.
-
+        """This method checks whether the given user has an admin role.
         This can be extended to groups as well for example a user belonging to certain group has only those
         privileges that are assigned to the group.
         """
+        if self.role == Roles(1):
```

Comment:
`Roles.ADMIN` is a lot easier to read, without having to find the ENUM to figure out which role is `1`. Don't use ENUMs as integers!